pkg/httpclient/client.go

```diff
@@ -241,12 +239,7 @@ func validateStatusCode(r *HTTPResponse, expectedStatuses []int, verb string, ur
 	useful.PanicOnUnrecoverableError(err)
 
 	// otherwise augment the error and return false
-	sb := strings.Builder{}
-	sb.WriteString(fmt.Sprintf("failed to execute %s request to %s\n", verb, url))
-	sb.WriteString(fmt.Sprintf("status code: %d\n", r.Response.StatusCode))
-	sb.WriteString(fmt.Sprintf("response: %s\n", r.Response.Status))
-	sb.WriteString(fmt.Sprintf("details: %s\n", errorDetails))
-	r.Err = errgo.Notef(r.Err, sb.String())
+	r.Err = fmt.Errorf("failed to execute %s request to %s\n status code: %d\n response: %s\n details: %s\n %w", verb, url, r.Response.StatusCode, r.Response.Status, errorDetails, r.Err)
```

Comment:
go 1.13 added this, so we can skip using `errgo` altogether since this was it's only use